### PR TITLE
Update the validations / regex for Japanese name to exclude half-width katakana.

### PR DESCRIPTION
**Pull request by RageZBla**
References #157

**Proposed Changes**

Update regular expression to exclude the half-width katakana range.

**Testing Plan**

Unit test provided.

### CODE REVIEW

**Comment by ann-kilzer** [comment]
Looking great. Let's also update the validation hint in en.json to state "Please enter your name in full width Kanji / Kana"

SpeakHer

**Comment by RageZBla** [comment]
@ann-kilzer thanks for your feedback, I have updated the validation messages.

File: web/src/i18n/locales/ja.json [hunk 1]
```diff
@@ -99,8 +99,8 @@
         "enNameRequired": "ローマ字で入力してください",
         "fieldRequired": "{0}が必要です",
         "invalidForm": "フォームに記入してください",
-        "jaNameCharacters": "漢字\/かなでお名前を入力してください",
-        "jaNameRequired": "漢字\/かなで入力してください",
+        "jaNameCharacters": "漢字\/全角かなでお名前を入力してください",
+        "jaNameRequired": "漢字\/全角かなで入力してください",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:103]: It appears the translation on this line has gotten out of sync with the english key. We should be translating "Japanese name is required" and don't need to mention the full-width or character types here.

**Review by ann-kilzer — COMMENTED** [review]
Super close, one last bit. Are you fluent? Otherwise we can ask our translation team to take a look.

**Comment by tuttiq** [comment]
全角 seems correct: https://en.m.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (RageZBla) [reply web/src/i18n/locales/ja.json]: @ann-kilzer thanks, I have updated the message.

**Review by tuttiq — APPROVED** [review]
LGTM! Can you update from master so we can merge it? 🙂

**Comment by ann-kilzer** [comment]
> 全角 seems correct: https://en.m.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms

Right, I was asking more about translating the out-of-sync phrase. All good it seems.

**Comment by RageZBla** [comment]
@tuttiq @ann-kilzer rebased 🎉

**Comment by ann-kilzer** [comment]
Please don't use force-push. It makes working with the branch harder for everyone else.

**Comment by tuttiq** [comment]
Thanks for writing the guidelines @ann-kilzer ! (I also didn't know about the preference to merge master over rebase+force push, but let's go with this then)

Merged 🚀

**Pull request merged**